src/index.ts

```diff
@@ -83,13 +108,13 @@ function getRelativeFormElement(element: HTMLInputElement, selector: string) {
         // Form.Password
         let relativeElementName = objectName + '.' + realSelector;
         let relativeElement = document.getElementsByName(relativeElementName)[0];
```

Comment:
Not new here, but I wonder if this should be constrained to `element.form` instead of `document`?

Reply:
Possibly. Seems out of scope for this PR though. I'd want to see a demo in `Index.cshtml` of the problem this causes.